images/miq-app-frontend/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM bdunne/miq-pods-test:backend-latest
```

Comment:
I think we'll need this changed before we merge.

Also, on that note, should we change the directory in this repo to `manageiq-app-backend`?